library/Zend/Http/Client/Adapter/Socket.php

```diff
@@ -357,7 +357,7 @@ public function read()
         }
 
         // If we got a 'transfer-encoding: chunked' header
-        if (isset($headers['transfer-encoding'])) {
+        if (isset($headers['transfer-encoding']) && is_string($headers['trasnsfer-encoding'])) {
```

Comment:
Typo in `trasnsfer`?

Reply:
Well spotted ... obviously a err deliberate error to check if someone reviewed this .. or something .... :hide: